src/Domain/Setting/SettingPersister.ts

```diff
@@ -0,0 +1,31 @@
+import { inject, injectable } from 'inversify'
+import TYPES from '../../Bootstrap/Types'
+import { CreateOrReplaceSettingDto } from './CreateOrReplaceSettingDto'
+import { CreateOrReplaceSettingStatus } from './CreateOrReplaceSettingStatus'
+import { SettingFactory } from './SettingFactory'
+import { SettingRepositoryInterface } from './SettingRepositoryInterface'
+
+@injectable()
+export class SettingPersister {
```

Comment:
missing `SettingPersisterInterface`

Reply:
hmm maybe `SettingService` and `SetttingServiceInterface` would also fit in terms of naming? 

Example: https://github.com/standardnotes/syncing-server-js/blob/develop/src/Domain/Item/ItemServiceInterface.ts

Reply:
Can change to SettingService (although Persister is more accurate IMO, Service can be anything).
Ad. interface: https://github.com/standardnotes/auth/pull/25#discussion_r630963890

Reply:
renamed